package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -183,84 +167,124 @@ class DistanceMatrix(AnalysisBase):
     using a given metric
 
     A distance matrix can be initialized on its own and used as an
-    initialization argument in :class:`DiffusionMap`. Refer to the
-    :ref:`Distance-Matrix-tutorial` for a demonstration.
+    initialization argument in :class:`DiffusionMap`.
+
+    Parameters
+    ----------
+    universe : `~MDAnalysis.core.universe.Universe`
+        The MD Trajectory for dimension reduction, remember that
+        computational cost of eigenvalue decomposition
+        scales at O(N^3) where N is the number of frames.
+        Cost can be reduced by increasing step interval or specifying a
+        start and stop value when calling :meth:`DistanceMatrix.run`.
+    select: str, optional
+        Any valid selection string for
+        :meth:`~MDAnalysis.core.groups.AtomGroup.select_atoms`
+        This selection of atoms is used to calculate the RMSD between
+        different frames. Water should be excluded.
+    metric : function, optional
+        Maps two numpy arrays to a float, is positive definite and
+        symmetric. The API for a metric requires that the arrays must have
+        equal length, and that the function should have weights as an
+        optional argument. Weights give each index value its own weight for
+        the metric calculation over the entire arrays. Default: metric is
+        set to rms.rmsd().
+    cutoff : float, optional
+        Specify a given cutoff for metric values to be considered equal,
+        Default: 1EO-5
+    weights : array, optional
+        Weights to be given to coordinates for metric calculation
+    verbose : bool, optional
+            Show detailed progress of the calculation if set to ``True``; the
+            default is ``False``.
 
     Attributes
     ----------
-    atoms : AtomGroup
+    atoms : `~MDAnalysis.core.groups.AtomGroup`
         Selected atoms in trajectory subject to dimension reduction
-    dist_matrix : array, (n_frames, n_frames)
+    results.dist_matrix : numpy.ndarray, (n_frames, n_frames)
         Array of all possible ij metric distances between frames in trajectory.
         This matrix is symmetric with zeros on the diagonal.
 
+        .. versionadded:: 2.0.0
+
+    dist_matrix : numpy.ndarray, (n_frames, n_frames)
+
+        .. deprecated:: 2.0.0
+                 Will be removed in MDAnalysis 3.0.0. Please use
+                 :attr:`results.dist_matrix` instead.
+
+    Example
```

Comment:
add the link target unless you know that this is not used anywhere 
```reST
.. _Distance-Matrix-tutorial:
```

Reply:
Thanks for the reminder. I searched the whole repo and didn't found any link left.